kentikapi/client_integration_test.go

```diff
@@ -252,12 +255,14 @@ func (s *spySyntheticsServer) Start() {
 	s.server = grpc.NewServer()
 	syntheticspb.RegisterSyntheticsAdminServiceServer(s.server, s)
 	go func() {
-		require.NoError(s.t, s.server.Serve(l))
+		err = s.server.Serve(l)
+		require.NoError(s.t, err)
+		s.done <- true
 	}()
 }
 
 func (s *spySyntheticsServer) Stop() {
-	s.server.GracefulStop()
+	s.server.Stop()
```

Comment:
What were the issues with GracefulStop?

Reply:
Stop always makes tests one second shorter because because it doesn't wait for the last response in the last test to return like GracefulStop() does. Since we don't need it I changed it to Stop().